### PR TITLE
Docker next release django 1.7

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: mamboms/mamboms/mambomsuser/__init__.py [hunk 1]
```diff
@@ -17,9 +17,9 @@ def create_django_user(user_name):
 
 def create_user_profile_for(django_user):
     try:
-        django_user.get_profile()
-    except SiteProfileNotAvailable:
-        print 'Profiles are not enabled for this app'
+        django_user.profile
+    # except SiteProfileNotAvailable:
```
Comment (aahunter) [inline mamboms/mamboms/mambomsuser/__init__.py:21]: If this code is dead best to remove it.

File: mamboms/setup.py [hunk 2]
```diff
@@ -40,7 +40,7 @@
         'pyparsing==1.5.6',
         'python-memcached==1.58',
         'wsgiref==0.1.2',
-        'South==1.0.2',
+        #'South==1.0.2',
```
Comment (aahunter) [inline mamboms/setup.py:43]: Why is south commented out?

**Comment by aahunter** [comment]
Firing up the app locally, the following warnings/errors are shown:

> runserver_1  | System check identified some issues:
> runserver_1  | 
> runserver_1  | WARNINGS:
> runserver_1  | ?: (1_6.W001) Some project unittests may not execute as expected.
> runserver_1  | 	HINT: Django 1.6 introduced a new default test runner. It looks like this project was generated using Django 1.5 or earlier. You should ensure your tests are all running & behaving as expected. See https://docs.djangoproject.com/en/dev/releases/1.6/#new-test-runner for more information.
> runserver_1  | mambomsapp.GCMARecord.known: (1_6.W002) BooleanField does not have a default value.
> runserver_1  | 	HINT: Django 1.6 changed the default value of BooleanField from False to None. See https://docs.djangoproject.com/en/1.6/ref/models/fields/#booleanfield for more information.
> runserver_1  | mambomsapp.LCMARecord.known: (1_6.W002) BooleanField does not have a default value.
> runserver_1  | 	HINT: Django 1.6 changed the default value of BooleanField from False to None. See https://docs.djangoproject.com/en/1.6/ref/models/fields/#booleanfield for more information.
> runserver_1  | Attempting to import local settings as appsettings.mamboms
> runserver_1  | Successfully imported appsettings.mamboms
> runserver_1  | Traceback (most recent call last):
> runserver_1  |   File "/env/bin/django-admin.py", line 5, in <module>
> runserver_1  |     management.execute_from_command_line()
> runserver_1  |   File "/env/lib/python2.7/site-packages/django/core/management/__init__.py", line 385, in execute_from_command_line
> runserver_1  |     utility.execute()
> runserver_1  |   File "/env/lib/python2.7/site-packages/django/core/management/__init__.py", line 377, in execute
> runserver_1  |     self.fetch_command(subcommand).run_from_argv(self.argv)
> runserver_1  |   File "/env/lib/python2.7/site-packages/django/core/management/base.py", line 288, in run_from_argv
> runserver_1  |     self.execute(*args, **options.__dict__)
> runserver_1  |   File "/env/lib/python2.7/site-packages/django/core/management/base.py", line 338, in execute
> runserver_1  |     output = self.handle(*args, **options)
> runserver_1  |   File "/env/lib/python2.7/site-packages/django_extensions/management/commands/update_permissions.py", line 20, in handle
> runserver_1  |     create_permissions(app, get_models(), int(options.get('verbosity', 0)))
> runserver_1  |   File "/env/lib/python2.7/site-packages/django/contrib/auth/management/__init__.py", line 62, in create_permissions
> runserver_1  |     if not app_config.models_module:
> runserver_1  | AttributeError: 'module' object has no attribute 'models_module'
>

**Comment by aahunter** [comment]
`django-admin check` call needs to be updated:

>   [INFO] running check --deploy
> uwsgi_1      | + django-admin.py check --deploy --settings=mamboms.settings
> uwsgi_1      | + tee /data/log/uwsgi-check.log
> uwsgi_1      | Usage: /env/bin/django-admin.py check [options] 
> uwsgi_1      | 
> uwsgi_1      | Checks the entire Django project for potential problems.
> uwsgi_1      | 
> uwsgi_1      | /env/bin/django-admin.py: error: no such option: --deploy
>

File: mamboms/mamboms/mambomsuser/__init__.py (hunk 1 above)
Comment (ghost) [reply mamboms/mamboms/mambomsuser/__init__.py:21]: Yep this is dead code. (https://code.djangoproject.com/ticket/19975). Removing.

File: mamboms/setup.py (hunk 2 above)
Comment (ghost) [reply mamboms/setup.py:43]: It was giving me an error and I found the following related to south: https://docs.djangoproject.com/en/1.7/topics/migrations/#upgrading-from-south
But I have uncommented and rebuilt it seems fine now.

Comment (ghost) [reply mamboms/setup.py:43]: See also: http://south.aeracode.org/

**Pull request closed without merging**